sass/_header.scss

```diff
@@ -169,6 +189,25 @@ body {
             padding: 0;
         }
 
+        .img-container {
+            margin-right: 0.25rem;
+            background-color: $secondary;
```

Comment:
Macht das Sinn, das in den Personal-Header mit aufzunehmen? Wird das nicht i.d.R. dafür verwendet, auch sein eigenes Logo zu verwenden?
![2024-04-17_15-45](https://github.com/verdigado/sunflower/assets/3354500/183f2446-2dc3-438d-9b16-44b1dbc4a5b3)
![2024-04-17_15-44](https://github.com/verdigado/sunflower/assets/3354500/95dfb9e3-eb4d-497b-a2ef-c0d8e7a77134)

Reply:
Wenn man bspw. bei Ricarda Lang schaut, dann ist dort auch die Sonnenblume. 
Personal wird auch für Mandatsträger:innen verwendet.

Reply:
Ok, wir haben aber z.B. auch sowas, wie bgkom.de. Da sähe es dann auch eher schräg aus.

![2024-04-17_16-09](https://github.com/verdigado/sunflower/assets/3354500/9bd8143a-87fc-4e82-8b5a-5e03728a3dee)

Mal überlegen, ob man das abhängig vom custom-logo machen kann.

Reply:
Man könnte z.B. im `header-personal.php` eine zusätzliche Klasse anbringen, wenn es ein Custom-Logo ist (bzw. anders rum - eine zusätzliche Klasse, wenn es kein Custom-Logo ist):

`<div class="img-container <?php if (! has_custom_logo()) echo 'sunflower-logo';?>">`

Reply:
eingebaut & noch ein Problem mit dem Logo nach der Einführung des Markenrechts-Hinweises beim Standard-Header gelöst. Bitte schau noch mal @HueHa-v